Helper/ConfigHelper.php

```diff
@@ -458,13 +458,18 @@ public function isAutocompleteDebugEnabled($storeId = null)
         return $this->configInterface->isSetFlag(self::AUTOCOMPLETE_MENU_DEBUG, ScopeInterface::SCOPE_STORE, $storeId);
     }
 
-    public function getSorting($storeId = null)
+    public function getSortingDefaultValue($storeId = null)
```

Comment:
Isn't it more a "raw" value than a "default" one ?

Reply:
Updated to:

getRawSortingValue
getRawProductCustomRanking